airflow/providers/google/cloud/example_dags/example_tasks.py

```diff
@@ -47,6 +46,7 @@
     CloudTasksTasksListOperator,
 )
 from airflow.utils.dates import days_ago
+from airflow.utils.helpers import chain
```

Comment:
@turbaszek I am wondering if we can update this in `dev/provider_packages/refactor_provider_packages.py`

Reply:
But I see this:

https://github.com/apache/airflow/blob/0fd5807632468ebdf349d47372b5226bc8160d36/dev/provider_packages/refactor_provider_packages.py#L601-L602

So I am not sure if I am missing something

Reply:
For now we can merge it as is to make Master green but I am curious if we can just modify that import using bowler